packages/main/src/FileUploader.js

```diff
@@ -374,7 +374,9 @@ class FileUploader extends UI5Element {
 			formData.append(this.name, this.files[i]);
 		}
 
-		this._internals.setFormValue(formData);
+		if (this._internals.setFormValue) {
+			this._internals.setFormValue(formData);
+		}
```

Comment:
This way the file is not send to the server when the containing form is submitted